firmware/RAD/release/Core/Src/main.c

```diff
@@ -821,6 +833,33 @@ int main(void)
                 {
                     rad_can.id = new_message->data[0];
                     rad_params.RAD_ID = rad_can.id;
+
+                    MX_CAN_UpdateIdAndFilters(&rad_can);
+
+                    break;
+                }
+                case SET_PID_ERROR_THRESHOLD:
+                {
+                    pid_1.Init.min_output_abs = (double) new_message->data[0];
+                    rad_params.PID_ERROR_THRESHOLD = pid_1.Init.min_output_abs;
+
+                    break;
+                }
+                case GET_PID_ERROR_THRESHOLD:
+                {
+                    MX_CAN_Broadcast_Uint8_Data(&rad_can, pid_1.Init.min_output_abs, GET_PID_ERROR_THRESHOLD);
+                    break;
+                }
+                case SET_PID_MAX_OUTPUT:
+                {
+                    pid_1.Init.max_output_abs = (double) decode_uint16_big_endian(new_message->data);
```

Comment:
Just a nitpick here but why does `max_output_abs` expect a `uint16` while `min_output_abs` is a `uint8` lol

Other than that, LGTM